tags/tags-api/api/src/java/org/sakaiproject/tags/api/i18n/tagservice_eu.properties

```diff
@@ -0,0 +1,2 @@
+site_collection=Colecci\u00f3n del sitio con id {0}, usadas en la herramienta de Tareas.
+user_collection=Colecci\u00f3n personal de etiquetas, usadas en la herramienta de Mensajes Privados.
```

Comment:
@PabloSanRoman  Sorry, those two lines have to be translated too. Thanks!

Reply:
Hi @bgarciaentornos !
site_collection={0} IDa duen gunearen bilduma, Zereginak tresnan erabiltzen ari direnak. 
user_collection=Etiketa pertsonalen bilduma, Emailak tresnan erabiltzen ari direnak.